chore: add 0.73.5 release summary notes (#829)
vegaprotocol/documentation: versioned_docs/version-v0.73/releases/overview.md

The Vega core software is public on a business-source licence, so you can both view the repository change logs, and refer here for summary release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.

### Release version 0.73.5 (patch) | 2023-11-20
Version 0.73.5 was released by the validators to mainnet on 20 November, 2023.

This release contains a number of critical fixes required as a result of testing feedback after the 0.73 deployment.

Check out the full details of what is contained in the patch release in the Vega core [0.73.5 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.73.5) release page.

### Critical fixes

> An issue was noticed affecting perpetual markets when the market doesn't have at least 1 of each price observation (internal and external). When this occurs the funding payment doesn't get exchanged, however, the Margin Estimate assumes it will happen and hence provides and incorrect estimate for the side due to make the payment. This issue was resolved in 3 pull requests [10119 ↗](https://github.com/vegaprotocol/vega/pull/10119), [10121 ↗](https://github.com/vegaprotocol/vega/pull/10121) and [10124 ↗](https://github.com/vegaprotocol/vega/pull/10124).

Author: ```suggestion
An issue was noticed affecting perpetual markets when the market doesn't have at least 1 of each price observation (internal and external). When this occurs the funding payment doesn't get exchanged, however, the Margin Estimate assumes it has exchanged and thus provides and incorrect estimate for the side due to make the payment. This issue was resolved in 3 pull requests [10119 ↗](https://github.com/vegaprotocol/vega/pull/10119), [10121 ↗](https://github.com/vegaprotocol/vega/pull/10121) and [10124 ↗](https://github.com/vegaprotocol/vega/pull/10124).
```

> During a [Market Simulation ↗](https://github.com/vegaprotocol/vega-market-sim) fuzzing test run when sending a batch market transaction a bug was identified where it crashed the network. This was found to be nn invalid stop order without a rises-above or falls-below and causing count of how many pre-generated IDs required to fall out of sync. This was resolved in the pull request [10070 ↗](https://github.com/vegaprotocol/vega/pull/10070)

Author: ```suggestion
During a [Market Simulation ↗](https://github.com/vegaprotocol/vega-market-sim) fuzzing test, when sending a batch market transaction, a bug was identified where it crashed the network. This was found to be an invalid stop order without a rises-above or falls-below that causes the count of how many pre-generated IDs required to fall out of sync. This was resolved in the pull request [10070 ↗](https://github.com/vegaprotocol/vega/pull/10070)
```

> When carrying out some testing of `StopOrdersSubmission` in batch proposals it was found that sending this proposal without any values crashes datanode. The investigation found this to be a nill pointer panic and this bug was fixed in the pull requests [10126 ↗](https://github.com/vegaprotocol/vega/pull/10126)

Author: ```suggestion
When carrying out some testing of `StopOrdersSubmission` in batch proposals it was found that sending this proposal without any values crashes the data node. The investigation found this to be a nill-pointer panic and this bug was fixed in the pull requests [10126 ↗](https://github.com/vegaprotocol/vega/pull/10126)
```
